pkg/sql/alter_function.go

```diff
@@ -86,7 +87,26 @@ func (n *alterFunctionOptionsNode) startExec(params runParams) error {
 		return err
 	}
 
-	return params.p.writeFuncSchemaChange(params.ctx, fnDesc)
+	if err := params.p.writeFuncSchemaChange(params.ctx, fnDesc); err != nil {
+		return err
+	}
+
+	fctx := tree.NewFmtCtx(tree.FmtSimple)
+	for i := range n.n.Options {
+		if i > 0 {
+			fctx.WriteString(", ")
+		}
+		fctx.FormatNode(n.n.Options[i])
+	}
+	fnName, err := params.p.getQualifiedFunctionName(params.ctx, fnDesc)
+	if err != nil {
+		return err
+	}
+	event := eventpb.AlterFunctionOptions{
+		FunctionName: fnName.FQString(),
+		Options:      fctx.CloseAndGetString(),
```

Comment:
adding `alter function` details here is probably a overkill....

Reply:
the event log json records the whole statement, so no need to book keeping redundant option details